res/skins/Deere/style.qss

```diff
@@ -1939,11 +1895,161 @@ do not highlight either state. */
  ** End Buttons  ***************************************************************
  *******************************************************************************/
 
-QToolTip {
-    background-color: #222222;
-    color: #CCCCCC;
-    border: 1px solid #CCCCCC;
+
+WEffectSelector QAbstractItemView,
+QToolTip,
+QMenu,
+QMenu QCheckBox {
+  padding: 2px;
+  font: 13px;
+}
+WEffectSelector QAbstractItemView {
+  min-width: 140px;
+}
+/* common colors for WEffectSelector, QMenu, QToolTip */
+WEffectSelector QAbstractItemView,
+WEffectSelector::indicator:!checked,
+QToolTip,
+QMenu,
+QMenu::item,
+QMenu QCheckBox {
+  color: #c1cabe;
+  background-color: #201f1f;
+  }
+WEffectSelector QAbstractItemView,
+QToolTip,
+QMenu {
+  border-width: 1px;
+  border-style: solid;
+  border-color: #aaa;
+  border-radius: 1px;
+  }
+  /* hovered items */
+  WEffectSelector::item:selected,
+  QMenu::item:selected,
+  QMenu QCheckBox:selected,
+  QMenu QCheckBox:focus,  /* selected by keyboard */
+  QMenu QCheckBox:hover  /* mouse hover */ {
```

Comment:
What i do think is, that at least on macOS, the styled menus are harder to read than before. Specially the active items. Try to look from some distance. With this fore-/ background colors, text has a color contrast ratio of 2.8:1, we should aim for at least 4.5  with such small fonts according to WCAG 
![](https://user-images.githubusercontent.com/4525897/68868280-8d90fe80-06f7-11ea-9164-d586440e3714.png "Deere master") ![](https://user-images.githubusercontent.com/4525897/68868306-97b2fd00-06f7-11ea-93be-1c8ecf874b71.png "Deere this PR")

Reply:
I see. On one hand, the pupose of this PR is to avoid those super bright context menus in our nighttime skins. Otoh I see the contrast can be increased for Deere.
How does the contrast feel like in the other skins? I think in LateNight at least it's sufficient consindering the overall contrast in the skin:
![image](https://user-images.githubusercontent.com/5934199/68879576-f08b9100-0709-11ea-8568-ff339f8e263c.png)

Reply:
Btw, thanks for testting!

Reply:
I think the contrast for hovered menu items is okay in the other skins. IMO it is only Deere where it is a problem.

Reply:
Please take care of this in a new PR since this one was merged prematurely.